Sources/Jobs/JobQueueHandler.swift

```diff
@@ -112,20 +115,36 @@ final class JobQueueHandler<Queue: JobQueueDriver>: Sendable {
                     self.updateJobMetrics(for: job.name, startTime: startTime, error: error)
                     return
                 } catch {
-                    if count <= 0 {
-                        logger.debug("Job failed")
+                    if job.didFail {
+                        logger.debug("Job: failed")
```

Comment:
Idea: Can we maybe add the job ID and/or typename to the logs?

Reply:
JobId is already attached in the logger metadata [here](https://github.com/hummingbird-project/swift-jobs/blob/main/Sources/Jobs/JobQueueHandler.swift#L75)